datadog-integrations-core.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: datadog-integrations-core
+  version: 7.53.0
+  epoch: 0
+  description: "Core integrations of the Datadog Agent"
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - krb5-dev
+      - linux-headers
+      - py3-pip
+      - python3-dev
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/DataDog/integrations-core
+      tag: ${{package.version}}
+      expected-commit: 052a2ee6820d3dc120b66f2a91d9eb22b00af01b
+
+  - runs: |
+      python3 -m pip install "./datadog_checks_base[deps, http]" --prefix=/usr --root=${{targets.destdir}}
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: DataDog/integrations-core
+    use-tag: true
```

Comment:
https://github.com/datadog/integrations-core uses releases. Should we omit `use-tag` so it pulls from the release instead?
```suggestion
```

Reply:
from what I can tell, the `ddev` releases are unrelated, what we actually want are the `x.y.z` tags, which map to the upstream agent version